src/pages/cli/restapi/override.mdx

```diff
@@ -152,3 +152,132 @@ user who is invoking the
 endpoint, and you can reuse the verified ID Token in your endpoint resolvers to assume
 the identity of the user for accessing other services like AWS AppSync or S3.
 
+## Authorize API requests with Lambda Authorizer
+
+In a similar way like Cognito Authorizer, Amplify CLI also does not support Lambda authorizers out-of-box. To implement this functionality, you must override your REST API and add a Lambda authorizer yourself by adding the following code into the `override(...)` function. The following steps are walk-through to create Token-Based Lambda Authorizer.
+
+First, assuming the Lambda Authorizer function you would like to use as an authorizer is configured with your Amplify Project, create a parameter that resolves to Lambda Function ARN:
```

Comment:
I'd rewrite this to also link to the flow where you run `amplify add function` and request customers to review there.

Reply:
Can you illustrate where I create this parameter? For example in xyz file... or "after you run...".

Reply:
Added, steps to create lambda function and mentioned file name to add the code